libraries/AP_HAL_ChibiOS/hwdef/FlywooF405S-AIO/hwdef.dat

```diff
@@ -152,5 +152,6 @@ define DEFAULT_NTF_LED_TYPES 257
 
 # save some flash space
 include ../include/no_bootloader_DFU.inc
+include ../include/minimize_fpv_osd.inc
```

Comment:
Please remove the no_bootloader_DFU.inc line at the same time. There is still 138k free flash with this which is plenty